api/client/proto/authservice.proto

```diff
@@ -1324,11 +1337,9 @@ service AuthService {
     rpc Ping(PingRequest) returns (PingResponse);
 
     // RotateResetPasswordTokenSecrets rotates token secrets for a given tokenID.
-    // DELETE IN: 9.0.0 in favor of RotateUserTokenSecrets.
+    // DELETE IN: 9.0.0 in favor of CreateRegisterChallenge.
     rpc RotateResetPasswordTokenSecrets(RotateUserTokenSecretsRequest)
         returns (types.UserTokenSecretsV3);
-    // RotateUserTokenSecrets rotates token secrets for a given tokenID.
-    rpc RotateUserTokenSecrets(RotateUserTokenSecretsRequest) returns (types.UserTokenSecretsV3);
```

Comment:
The rename from `RotateReset...` to `RotateUserToken` has not been backported yet, so I just removed the rename.